test/TEST-64-UDEV-STORAGE/test.sh

```diff
@@ -12,7 +12,6 @@ TEST_DESCRIPTION="systemd-udev storage tests"
 TEST_NO_NSPAWN=1
 # Save only journals of failing test cases by default (to conserve space)
 TEST_SAVE_JOURNAL="${TEST_SAVE_JOURNAL:-fail}"
-QEMU_TIMEOUT="${QEMU_TIMEOUT:-600}"
```

Comment:
This might affect manual runs, and runs that we don't know about - what about keeping this and adding it to all other instances? So that if it's set by the CI we use it, but there's a default if not?

Reply:
actually just now saw the next commit, that works just as fine